config/locales/pt-BR.yml

```diff
@@ -18,13 +18,13 @@ pt-BR:
       trackbacks_are_disabled: Trackbacks foram desabilitados
       trackbacks_for: Trackbacks para
     search:
-      search_results_for: Search results for '%{query}'
+      search_results_for: Buscar resultados para '%{query}'
```

Comment:
My knowledge of Portuguese is very sketchy, but:

'Search' is part of the compound noun 'search results' here, so I think this should be something like 'Resultados de busca para ...'.